ttnn/cpp/ttnn/operations/embedding/device/embedding_program_factory.hpp

```diff
@@ -92,63 +106,72 @@ operation::ProgramWithCallbacks embeddings_tilized(
     tt::DataFormat output_cb_data_format = tt_metal::datatype_to_dataformat_converter(output.get_dtype());
     uint32_t output_single_tile_size = tt_metal::detail::TileSize(output_cb_data_format);
 
-    uint32_t buffering = weights.get_legacy_shape()[-1] > 2048 ? 1 : 2;
+    uint32_t buffering = num_tiles_per_block > 64 ? 1 : 2;
```

Comment:
Please use a constexpr with a good description for 64.

Reply:
Done